english/light-fire/graham-crackers/chocolate/make-chocolate/make-chocolate.md

```diff
@@ -18,3 +18,4 @@ You are eaten by a grue.
 
 You die...
 
+[What is a grue?](../../../grue/grue.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.